assets/base/i18n/en.json

```diff
@@ -1291,6 +1291,7 @@
   "channel_header.rename": "Rename Channel",
   "channel_header.setHeader": "Edit Channel Header",
   "channel_header.setPurpose": "Edit Channel Purpose",
+  "channel_header.edit": "Edit Channel",
```

Comment:
If this string does not exist in the webapp then you need to add the **mobile.** prefix, the same goes with the ones below

Reply:
👍